<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminLogin</class>
 <widget class="QDialog" name="AdminLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#AdminLogin{
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #320453, stop:1 #802CBD);
}
QGroupBox#groupBox {
    background-color: #7F00FF;
    border-radius: 50px;
}

QLineEdit {
    background-color: #cecece;
    border-radius: 25px;
	font-size:16px;
	color:#3a3a3a;
	padding-left: 15px;
}
QLineEdit::placeholder{
	color: #3d3d3d;	
}

QPushButton#loginButton {
    background-color: #1DB500;
    border-radius: 20px;
    color: white;
    font-weight: bold;
}

QPushButton#loginButton:hover {
    background-color: #308420;
}
QDialog#StudentLogin {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #320453, stop:1 #802CBD);
}

QGroupBox#groupBox {
    background-color: #7F00FF;
    border-radius: 50px;
}

QLineEdit {
    background-color: #cecece;
    border-radius: 25px;
	font-size:16px;
	color:#3a3a3a;
	padding-left: 15px;
}
QLineEdit::placeholder{
	color: #3d3d3d;	
}

QPushButton#loginButton {
    background-color: #1DB500;
    border-radius: 20px;
    color: white;
    font-weight: bold;
}

QPushButton#loginButton:hover {
    background-color: #308420;
}
</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>521</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="usernamelineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>admin</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordlineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>1234</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LOGIN AS ADMIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>User Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>551</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ENSIA Library Management system</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
